qiskit_optimization/applications/tsp.py

```diff
@@ -167,7 +163,7 @@ def parse_tsplib_format(filename: str) -> "Tsp":
                     typ.strip()
                     if typ != "TSP":
                         raise QiskitOptimizationError(
-                            'This supports only "TSP" type. Actual: {}'.format(typ)
+                            f'This supports only "TSP" type. Actual: {typ}'
```

Comment:
I thought that via black strings ended up being in double quotes so a bit surprised to see this in single quotes along with the same thing a but further down this file

Reply:
Black ignored because the TSP constant already has double-quotes. It doesn't touch strings like that. There are other cases.